iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/facade/ObjectStoreFacade.java

```diff
@@ -0,0 +1,19 @@
+package eu.dnetlib.iis.wf.importer.facade;
+
+/**
+ * ObjectStore service facade.
+ * 
+ * @author mhorst
+ *
+ */
+public interface ObjectStoreFacade {
+
+    /**
+     * Provides all metadata records from given objectstore. 
```

Comment:
From params we can deduce that not `all metadata` has to be provided.
I would add info about time range or remove word `all`